src/Symfony/Component/Routing/Loader/XmlFileLoader.php

```diff
@@ -239,6 +241,10 @@ private function parseConfigs(\DOMElement $node, $path)
             $defaults['_controller'] = $controller;
         }
 
+        if ($redirectTo = $node->getAttribute('redirect-to')) {
```

Comment:
should forbid to put it at the same time than a controller (check for `$defaults['_controller']` being configured already)

Reply:
Done 👍